pkg/cluster/database.go

```diff
@@ -154,7 +154,10 @@ func (c *Cluster) initDbConnWithName(dbname string) error {
 				return false, err2
 			}
 
-			return false, err
+			// Ping verifies a connection to the database return EOF error.
```

Comment:
```suggestion
```

I find the wording of the first comment line confusing. Ping should verify that the connection will work, right. EOF is returned if Ping fails. Anyway, let's remove this line because Ping is not called here but above. The code with the new warning message is clear enough for me already. The // Retry open comment is fine with me.